### PR TITLE
Updated Polish (pl) localization of Firefox Health Report

### PR DESCRIPTION
**Pull request by piotrdrag**

### CODE REVIEW

**Comment by piotrdrag** [comment]
@splewako

File: pl/fhr.lang [hunk 1]
```diff
@@ -51,21 +51,21 @@ Wersja
...
 # Possible value for "updates"
 ;automatic
-automatic
+Automatyczne
 
 
 # Possible value for "updates"
 ;prompt
-prompt
+Pytanie użytkownika
```
Comment (splewako) [inline pl/fhr.lang]: Wouldn't "Wymagane potwierdzenie" be better here?

File: pl/fhr.lang [hunk 2]
```diff
@@ -747,12 +747,12 @@ Obecnie nie udostępniasz Mozilli swojego raportu o zdrowiu.
 
 # This string is displayed when selecting Raw Data near the title
 ;Firefox Health Report (FHR) records anonymous data about your computer, configuration, usage, performance, and error conditions. This data helps us improve Firefox for you and for other Firefox users.
-Firefox Health Report (FHR) records anonymous data about your computer, configuration, usage, performance, and error conditions. This data helps us improve Firefox for you and for other Firefox users.
+Raport o zdrowiu Firefoksa zbiera anonimowe dane o twoim komputerze, konfiguracji, użyciu przeglądarki, wydajności i błędach. Pomaga nam to ulepszać Firefoksa dla wszystkich użytkowników.
```
Comment (splewako) [inline pl/fhr.lang]: **T**woim

**Comment by splewako** [comment]
It wouldn't hurt also to `sed -E -e '/^[^#;]/ s/([  ])([aiouwz]) /\1\2 /g' -i '' filePath` (put hard space after single letter words preceded and followed by space).

**Comment by piotrdrag** [comment]
```
$ sed -E -e '/^[^#;]/ s/([  ])([aiouwz]) /\1\2 /g' -i '' fhr.lang 
sed: can't read : No such file or directory
```

I know absolutely nothing about sed.

**Comment by splewako** [comment]
I will deal with that later.

(done: https://github.com/mozilla-l10n/fhr-l10n/commit/a1f426effc81fa3589e65b8dde8b5b59e6785bdc)

**Comment by piotrdrag** [comment]
Thanks!

**Pull request merged**